Sources/ProjectDescription/TargetReference.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public struct TargetReference: Equatable, Codable, ExpressibleByStringLiteral {
+    public var projectPath: Path?
+    public var targetName: String
+
+    public init(projectPath: Path?, target: String) {
+        self.projectPath = projectPath
+        targetName = target
+    }
+
+    public init(stringLiteral value: String) {
+        self = .init(projectPath: nil, target: value)
+    }
+
+    public static func project(path: Path, target: String) -> TargetReference {
```

Comment:
wondering if we need this `static` builder function?

Reply:
You are right. I think is a leftover from copying and pasting the model from `ProjectDescription`. I'll remove it :+1: